data/fa

```diff
@@ -0,0 +1,26 @@
+---
+version: 2
+anglicized_name: Persian
+native_name: فارسی
+codepoints:
+- 171                    # «
+- 187                    # »
+- 1548                   # ،
+- 1563                   # ؛
+- 1567                   # ؟
+- !ruby/range 1569..1572 # ء,آ,أ,ؤ
+- !ruby/range 1574..1594 # ئ,ا,ب,ة,ت,ث,ج,ح,خ,د,ذ,ر,ز,س,ش,ص,ض,ط,ظ,ع,غ
+- !ruby/range 1601..1602 # ف,ق
+- !ruby/range 1604..1608 # ل,م,ن,ه,و
+- !ruby/range 1611..1613 # \u064b,\u064c,\u064d
+- 1617                   # \u0651
+- 1620                   # \u0654
+- !ruby/range 1643..1644 # ٫,٬
+- 1662                   # پ
+- 1670                   # چ
+- 1688                   # ژ
+- 1705                   # ک
+- 1711                   # گ
+- 1740                   # ی
+- 8209                   # ‑
```

Comment:
The 8209 is U+2011 NON-BREAKING HYPHEN, and I think we should not require it for a language match (and we already exclude other kinds of hyphens from the data).  I'll update the script.

Reply:
Thanks - updated to remove these